shader/main3d.frag.glsl

```diff
@@ -168,13 +170,14 @@ void main()
   // If those where not used, a race-condition will occur where after a depth read happens, the depth value might have changed,
   // leading to culled faces writing their color values even though a new triangles has a closer depth value already written.
   ivec2 screenPosPixel = ivec2(gl_FragCoord.xy);
-  int currDepth = int(gl_FragCoord.w * 0xFFFFF);
+
+  int currDepth = int(mixSelect(usePrimDepth(), gl_FragCoord.w * 0xFFFFF, primDepth.x));
```

Comment:
primDepth would be broken in blender right now since i can't fully change the depth value.
I'm forced to have proper depth compare enabled and use actual depth values to not overdraws blenders viewport (which it draws before me).
So it will not work in general until they allow me to have my own custom depth pass.

But i'd say it should stay in the shader since it doesn't affect any materials that don't use it.

Reply:
Yeah I noticed that it didn´t work but wanted to leave the implementation there just in case, delta z does work tho!